ui/src/app/pages/analysis/util.ts

```diff
@@ -20,14 +20,14 @@ export function appendNotebookFileSuffix(filename: string) {
   return filename;
 }
 
-export const getExistingNotebooks = (workspace): Promise<FileDetail[]> => {
```

Comment:
"existing" is a little weird here.  "list notebooks" is an accurate description and I think more intuitive.